build.gradle

```diff
@@ -15,5 +16,6 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        google()
```

Comment:
Same as above: this should probably be the first one.

Reply:
👍